pyntc/devices/nxos_device.py

```diff
@@ -362,7 +362,9 @@ def set_boot_options(self, image_name, kickstart=None, **vendor_specifics):
             kickstart = file_system + kickstart
 
         image_name = file_system + image_name
-        self.native.timeout = 300
+        # Allow for user defined timeout to take precedence if its over 300 seconds, otherwise change to 300.
+        if self.native.timeout < 300:
```

Comment:
what if self.native.timeout isn't an int?

Reply:
This has been updated @jmcgill298